### ISSUE TITLE
Missing error or warning on dconjg misuse

### ISSUE THREAD

**Issue by sscalpone** [flang-compiler/f18#661]
```
! gfortran always issues an error
! if INTRINSIC is used, ifort issues an error
! if INTRINSIC is commented out, ifort assumes EXTERNAL
! pgfortran says "Type conversion of expression performed"
! f18 is slient; still a todo
subroutine test(a)
  complex(4) :: a
  complex(4) :: c
  INTRINSIC dconjg
  c = dconjg(a)
end
```

**Comment by jeanPerier** [comment]
Fix by PR #680.
f18 now outputs error:
```
error: Actual argument for 'a=' has bad type or kind 'Complex(4)'
    c = dconjg(a)
        ^^^^^^
```

**Comment by sscalpone** [comment]
It took me a minute to understand the "=" in "a=".
